ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java

```diff
@@ -395,6 +387,33 @@ public void run() {
         var contractAccount = updater.getOrCreate(contract);
         contractAccount.setCode(codeBytes);
 
+        final Set<Address> addressList = new BytesTrieSet<>(Address.SIZE);
+        addressList.add(sender);
+        addressList.add(contract);
```

Comment:
What does this change mean in this PR?

Reply:
EIP-2929 says the sender and contract address are pre-warmed.  The transaction handler does that, but the evm tool has it's own bootstrap logic and needs to pre-warm them.